<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_logar</class>
 <widget class="QDialog" name="Fm_logar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Usu√°rio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_user">
        <property name="text">
         <string>gene</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Senha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_pass">
        <property name="text">
         <string>123</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Sair</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_login">
        <property name="text">
         <string>Logar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
